src/components/PageBody/PageBody.tsx

```diff
@@ -141,14 +141,12 @@ export function PageBody(props: {
                     ) : null}
                 </div>
             </main>
-            <React.Suspense fallback={null}>
```

Comment:
Removed the Suspense as I didn't see why it was necessary. Let me know if I missed something.

Reply:
It should be fine, I'm not sure if Next suspends when loading a client component